Src/BootCamp.Chapter/Checks.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Globalization; // NumberStyles.Float, CultureInfo.InvariantCulture
+
+class Lesson3
+{
+	public static void Demo()
+	{
+		string userName = "", userSurname = "";
+		int userAge = 0;
+		float userWeight = 0f, userHeight = 0f;
+
+		CheckForUserInput("Please input your name.", ref userName);
+		CheckForUserInput("Please input your surname.", ref userSurname);
+
+		Console.WriteLine("Hi " + userName + " " + userSurname + "!");
+
+		CheckForUserInput("Please input your age.", ref userAge);
+
+		CheckForUserInput("Please input your weight in kg.", ref userWeight);
+		CheckForUserInput("Please input your height in cm.", ref userHeight);
+
+		Console.WriteLine("Calculating BMI for " + userName + "...");
+		float bmi = userWeight / (userHeight * userHeight);
+		Console.WriteLine("BMI: " + bmi);
+	}
+
+	static void CheckForUserInput(in string outputText, ref string userInput)
+	{
+		bool validInput = false;
+
+		while (!validInput)
+		{
+			Console.WriteLine(outputText);
+			userInput = Console.ReadLine().Trim();
+			validInput = userInput.Length > 0;
+
+			if (validInput) break;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4859834)